addon/ng2/models/webpack-config.ts

```diff
@@ -40,20 +45,34 @@ export class NgCliWebpackConfig {
     }
 
     this.generateConfig();
-    this.config.plugins.unshift(new NgCliEnvironmentPlugin({
+    this.config[0].plugins.unshift(new NgCliEnvironmentPlugin({
       path: path.resolve(this.ngCliProject.root, `./${sourceDir}/app/environments/`),
       src: 'environment.ts',
       dest: `environment.${this.environment}.ts`
     }));
+
+    const assetsPluginInstance = new AssetsPlugin({ filename: 'cli.assets.json' });
```

Comment:
It temporarily stores all the assets in a file so I know which files needs to be included.
Please see https://webpack.github.io/docs/long-term-caching.html